File: ParadigmaComp/15Questoes/golang/questao13.go
package main
//Métodos Estáticos
import "fmt"

// Função para calcular o fatorial de um número
func Fatorial(n int) int {
	if n == 0 || n == 1 {
		return 1
	}
	return n * Fatorial(n-1)
}

func main() {
	fmt.Println(Fatorial(5))  // Fatorial de 5
}

File: ParadigmaComp/15Questoes/python/questao13.py
# Métodos Estáticos
class Matematica:
    @staticmethod
    def fatorial(n):
        if n == 0 or n == 1:
            return 1
        else:
            return n * Matematica.fatorial(n - 1)

# Exemplo de uso
print(Matematica.fatorial(5))  # Fatorial de 5
